tsl/src/continuous_aggs/create.c

```diff
@@ -2341,6 +2463,11 @@ cagg_create(const CreateTableAsStmt *create_stmt, ViewStmt *stmt, Query *panquer
 	bool finalized = DatumGetBool(with_clause_options[ContinuousViewOptionFinalized].parsed);
 
 	finalqinfo.finalized = finalized;
+	if (list_length(panquery->jointree->fromlist) > 1 && !materialized_only)
+	{
+		materialized_only = true;
+		elog(INFO, "For cagg with joins, materialised_only is updated to true");
```

Comment:
I think this should be `materialized_only`for consistency

Reply:
I guess this is a leftover and should be removed!